src/Utils/PricingUtils.ts

```diff
@@ -279,16 +280,47 @@ export function getEstimatedSpendAcknowledgeString(
       )} - ${currencySign}${calculateEstimateNumber(monthlyPrice)} monthly cost for the throughput above.`;
 }
 
-export function getUpsellMessage(serverId: string = "default", isFreeTier: boolean = false): string {
+export function getUpsellMessage(
+  serverId: string = "default",
+  isFreeTier: boolean = false,
+  isServerless: boolean = false
+): string {
   if (isFreeTier) {
     return `With free tier discount, you'll get the first 400 RU/s and 5 GB of storage in this account for free. Charges will apply if your resource throughput exceeds 400 RU/s.`;
-  } else {
-    let price: number = Constants.OfferPricing.MonthlyPricing.default.Standard.StartingPrice;
+  }
 
-    if (serverId === "mooncake") {
-      price = Constants.OfferPricing.MonthlyPricing.mooncake.Standard.StartingPrice;
-    }
+  if (isServerless) {
+    return `With serverless, your collection will only be able to support 5k RU/s traffic and 50GB of storage at max.`;
```

Comment:
Reverting this because according to the PM we don't need to show any message for serverless accounts. I have a separate branch where I'm trying to convert these panes to react.